liberty-maven-plugin/src/it/kernel-install-feature-test/pom.xml

```diff
@@ -37,7 +37,9 @@
         <module>install-features-pluginlist-it</module>
         <module>install-features-server-already-installed-lowercase-it</module>
         <module>install-features-server-it</module>
-        <module>install-ol-features-no-accept-license-it</module>
+        <module>install-ol-features-no-accept-license-it</module> 
+        <module>prepare-feature-it</module> 
```

Comment:
Is `prepare-feature` only valid for OL? Does anything prevent it from being configured and called for a WL installation?

Reply:
No it can be installed to WL as well. This section lists modules that can be installed to WL or OL.